web/docs/guides/database/json.mdx

```diff
@@ -1,21 +1,18 @@
 ---
 id: json
-title: "Working With JSON Data"
-description: How to use JSON Data in PostgreSQL and the Supabase Javascript API.
+title: "JSON"
+description: Using the JSON data type in PostgreSQL.
```

Comment:
I tidied up this pages a bit but then decided not to use it. We should come up with a "standard" format for these guides I think (similar to the FTS guide)